aspnetcore/security/anti-request-forgery/samples/6.x/AntiRequestForgerySample/Snippets/Program.cs

```diff
@@ -33,4 +33,19 @@ public static void AddAntiforgeryOptionsJavaScript(WebApplicationBuilder builder
         builder.Services.AddAntiforgery(options => options.HeaderName = "X-XSRF-TOKEN");
         // </snippet_AddAntiforgeryOptionsJavaScript>
     }
+
+    public static void AddAntiforgeryEndpoint(WebApplication app)
+    {
+#region snippet_AddAntiforgeryEndpoint
```

Comment:
No regions please!

Reply:
Sorry, I was getting a strange doc build issue and that snippet wasn't showing up. I'll revert.

Reply:
@Rick-Anderson how did you solve the issue? I was having the same issue here https://github.com/dotnet/AspNetCore.Docs/blob/1d5b3ff4162f87bf350a30ae3c1b16962465a513/aspnetcore/security/anti-request-forgery/samples/7.x/AntiRequestForgeryMinimalSample/Program.cs#L15

Reply:
I closed and opened the PR. There was a problem with the build system for one day.

Reply:
@Rick-Anderson I tried to remove the region but still not working. Is that anything else I need to? https://github.com/dotnet/AspNetCore.Docs/pull/26386/commits/bde4341f16891e77a233c9f07f3f3972168affe0